README.md

```diff
@@ -13,10 +13,15 @@
 - In case of setting up synchronization, no whitespace is allowed in the path to the project directory.
 
 # Setup
-### For Ubuntu,
+### For Unix-like OS
 1. Do
 ```
+## Ubuntu/Debian
 sudo apt install python3-dev libpq-dev
+
+## RPM-based Linux
```

Comment:
if we are only testing and communicating things about Fedora, then the section title should be Fedora, rather than the more generic _RPM-based_ term. Or it could be, with the same title, but add a disclaimer mentioning that we only tested Fedora, but the process should be similar in other derivations.

Reply:
How about adding the commet: 
Now we only tested on Fedora 31 successfully, but there should be no differences on other RPM-based Linux systems. If you can test on centOS/RedHat/OpenSuse and other versions of Fedora, please give us some feedback.

Reply:
Yeah, it looks good. But I would say "but there should be very little differences on other..."